Git.hub/OAuth2Helper.cs

```diff
@@ -50,13 +50,13 @@ public static string requestToken(string client_id, string client_secret, string
     /// (not even sure why) Using RestSharp's authenticators works on GET /user, but not GET /user/repos?
     /// This basically works around that.
     /// </summary>
-    class OAuth2AuthHelper : OAuth2Authenticator
+    class OAuth2AuthHelper : AuthenticatorBase
     {
         public OAuth2AuthHelper(string token) : base(token) { }
 
-        public override void Authenticate(IRestClient client, IRestRequest request)
-        {
-            request.AddHeader("Authorization", "bearer " + AccessToken);
+        protected override Parameter GetAuthenticationParameter(string accessToken)
+        {
+            return new Parameter("Authorization", "bearer " + Token, ParameterType.HttpHeader);

```

Comment:
Just a question. Is there a reason to use `Token` instead of the parameter `accessToken` that seems to contains the same value?
If not, for readability, I would use `accesToken` instead...

Reply:
I think it does the same thing and agree `accessToken` is more readable